libs/viewer/include/viewer/SimpleViewer.h

```diff
@@ -236,13 +236,15 @@ class UTILS_PUBLIC SimpleViewer {
     std::function<void()> mCustomUI;
 
     // Properties that can be changed from the UI.
-    int mCurrentAnimation = 1;
+    int mCurrentAnimation = 0;
```

Comment:
Personally I kind of like the auto-play feature that we had.  We could keep this set to `1` if we were to change the conditional as I've described in my subsequent comment.

By the way, we should probably have a comment here that explains that it is a 1-based index and 0 means "not playing animation".  That's my fault.

Reply:
Ok it's reverted.